src/main/java/com/powsybl/forcelayout/ForceLayout.java

```diff
@@ -124,9 +128,25 @@ public ForceLayout<V, E> setSpringRepulsionFactor(double springRepulsionFactor)
         return this;
     }
 
+    public ForceLayout<V, E> setInitialPoints(Map<V, Point> initialPoints) {
+        this.initialPoints = Objects.requireNonNull(initialPoints);
+        return this;
+    }
+
+    public ForceLayout<V, E> setFixedNodes(Set<V> fixedNodes) {
+        this.fixedNodes = Objects.requireNonNull(fixedNodes);
+        return this;
+    }
```

Comment:
These two methods should be in the `Layout` interface.

In fact I'm wondering if we could merge them by adding a fixed parameter in `Point`. Or by adding a `FixedPoint` class above `Point` which does not have velocity nor forces.

Reply:
The generic `Layout` interface points only have `(x,y)` coordinates.
The points inside the `forcelayout` package have the additional data about velocity and forces. Yes, we could add `FixedPoint` inside force layout to express this. Maybe in a separate PR ?